### PR TITLE
BP-105 Adding docs for new business APIs

### PR DESCRIPTION
**Pull request by kdowbecki**
Describing director, ubos and update-window APIs

### CODE REVIEW

File: source/includes/_banks.md [hunk 1]
```diff
@@ -282,6 +282,11 @@ Creating a business profile is similar to how you created personal profile. Ther
 
 2) [Create business user profile – address data](#addresses-create) 
 
+3) [Create business user profile - directors data](#user-profiles-add-business-directors)
+
+3) [Create business user profile - owners data](#user-profiles-add-business-ultimate-beneficial-owners)
+
+
 Currently we do not support a fully automated business onboarding flow over the API - this feature is coming soon, please contact the TransferWise team for more details.
```
Comment (JamesFerretti) [inline source/includes/_banks.md]: remove this message?

File: source/includes/reference/_profiles.md [hunk 2]
```diff
@@ -327,3 +327,354 @@ issuerCountry            | Issued by country code. For example "US"    | Text
...
+
+> Example Response:
+
+```json
+[
+  {
+    "id": <director id>,
+    "firstName": "John",
+    "lastName": "Doe",
+    "dateOfBirth": "1982-05-20",
```
Comment (JamesFerretti) [inline source/includes/reference/_profiles.md]: Make this more obviously fake data - e.g. `01-01`, security ops told me before to make sure our example data is obviously fake.

File: source/includes/reference/_profiles.md [hunk 3]
```diff
@@ -327,3 +327,354 @@ issuerCountry            | Issued by country code. For example "US"    | Text
...
+> Example Response:
+
+```json
+[
+    {
+        "id": <shareholder id>,
+        "name": "John Doe",
+        "dateOfBirth": "1982-05-20",
+        "countryOfResidenceIso3Code": "usa",
+        "addressFirstLine": "87 West Central St",
```
Comment (JamesFerretti) [inline source/includes/reference/_profiles.md]: Use more obviously fake data here too please

File: source/includes/reference/_profiles.md [hunk 4]
```diff
@@ -327,3 +327,354 @@ issuerCountry            | Issued by country code. For example "US"    | Text
...
+     -H "Authorization: Bearer <your api token>" 
+```
+
+> Example Response:
+
+```json
+
+```
+
+Opens the update window for updating the business profile information: details, directors, owners.
```
Comment (JamesFerretti) [inline source/includes/reference/_profiles.md]: This is not just business right? Also this endpoint will only be for banks, maybe we should have separate docs? Not sure how to handle this.

File: source/includes/reference/_profiles.md [hunk 5]
```diff
@@ -327,3 +327,354 @@ issuerCountry            | Issued by country code. For example "US"    | Text
...
+     -H "Authorization: Bearer <your api token>" 
+```
+
+> Example Response:
+
+```json
+
+```
+
+Deletes the update window for updating the business profile.
```
Comment (JamesFerretti) [inline source/includes/reference/_profiles.md]: Personal too I think?

File: source/includes/_banks.md (hunk 1 above)
Comment (kdowbecki) [reply source/includes/_banks.md]: The user needs to provide all the information about the profile, including directors and shareholders before we can make a transfer. Am I incorrect and we don't need shareholders?

File: source/includes/reference/_profiles.md (hunk 2 above)
Comment (kdowbecki) [reply source/includes/reference/_profiles.md]: Done

File: source/includes/reference/_profiles.md (hunk 3 above)
Comment (kdowbecki) [reply source/includes/reference/_profiles.md]: Changed address and post code to `123 Fake St` and `FK 12345`

File: source/includes/reference/_profiles.md (hunk 5 above)
Comment (kdowbecki) [reply source/includes/reference/_profiles.md]: Done.

File: source/includes/_banks.md (hunk 1 above)
Comment (kdowbecki) [reply source/includes/_banks.md]: Done

File: source/includes/reference/_profiles.md (hunk 4 above)
Comment (kdowbecki) [reply source/includes/reference/_profiles.md]: Left as-is for now. The endpoint is only referenced in bank section and calling it with non-bank token throws an exception in rest gateway.

**Pull request merged**